### PR TITLE
hardening/490_update_qsg

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #490 
* No unit tests not e2e tests are required.
* Assignee @fgalan

### CODE REVIEW

File: doc/quick_start_guide.md [hunk 1]
```diff
@@ -13,14 +13,16 @@ Cygnus uses the subscription/notification feature of Orion. A subscription is ma
 ###Which storages is it able to integrate?
 Current stable release is able to persist Orion context data in:
 
-* HDFS, the Hadoop distributed file system.
-* MySQL, the well-know relational database manager.
-* CKAN, an Open Data platform.
+* [HDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html), the [Hadoop](http://hadoop.apache.org/) distributed file system.
+* [MySQL](https://www.mysql.com/), the well-know relational database manager.
```
Comment (fgalan) [inline doc/quick_start_guide.md]: well-know -> well-known

File: doc/quick_start_guide.md [hunk 2]
```diff
@@ -72,8 +74,8 @@ cygnusagent.sources.http-source.handler.default_service_path = def_servpath
 cygnusagent.sources.http-source.handler.events_ttl = 2
 cygnusagent.sources.http-source.interceptors = ts de
 cygnusagent.sources.http-source.interceptors.ts.type = timestamp
-cygnusagent.sources.http-source.interceptors.de.type = com.telefonica.iot.cygnus.interceptors.DestinationExtractor$Builder
-cygnusagent.sources.http-source.interceptors.de.matching_table = /Applications/apache-flume-1.4.0-bin/conf/matching_table.conf
+cygnusagent.sources.http-source.interceptors.de.type = com.telefonica.iot.cygnus.interceptors.GroupingInterceptor$Builder
+cygnusagent.sources.http-source.interceptors.de.matching_table = /Applications/apache-flume-1.4.0-bin/conf/grouping_rules.conf
```
Comment (fgalan) [inline doc/quick_start_guide.md]: Configuration parameters have changed... potential backward compability problem?

Comment (frbattid) [inline doc/quick_start_guide.md]: Yes, but there is no easy way to fix it: the `com.telefonica.iot.cygnus.interceptors.GroupingInterceptor$Builder` is the name of a class, and it is automatically taken by Apache Flume to instantiate it... there is no code controlled by us allowing to say "if appearing the old class name, instantiate the new one instead".

BTW, I've noticed there is another change in the configuration, I'll upload another commit.

File: doc/quick_start_guide.md (hunk 1 above)
Comment (frbattid) [inline doc/quick_start_guide.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5f3cda70057fb35842696996e986ba62f4a72e90

File: doc/quick_start_guide.md (hunk 2 above)
Comment (frbattid) [inline doc/quick_start_guide.md]: New change on the configuration appearing in https://github.com/telefonicaid/fiware-cygnus/commit/5f3cda70057fb35842696996e986ba62f4a72e90

Comment (fgalan) [inline doc/quick_start_guide.md]: So you mean that the change in the .conf is just a side-effect of chaning class names due to a refactor in the code?

Comment (frbattid) [inline doc/quick_start_guide.md]: Yes.

Comment (fgalan) [inline doc/quick_start_guide.md]: NTC

**Comment by fgalan** [comment]
LGTM (but I cannot merge as github points out a conflict)

**Pull request merged**